README.md

```diff
@@ -16,15 +16,16 @@ It's aims to be as fast and simple as possible.
 ## Setup
 
 - Download the latest version from `releases`
-- Create a `headers.txt` file and copy your headers from the nav when browsing https://music.youtube.com/
-  - Open the YouTube Music website in your browser");
-  - Open the developer tools (F12)
-  - Go to the Network tab
-  - Go to https://music.youtube.com
-  - Copy the `cookie` header from the associated request
-  - Paste it in the `headers.txt` file as `Cookie: <cookie>`
-  - Restart YterMusic
-- Run `ytermusic.exe`
+- Give `ytermusic` authentication to your account, by copying out the headers
+  1. Open the https://music.youtube.com website in your browser
+  2. Open the developer tools (F12)
+  3. Go to the Network tab
+  4. Find the request to the `music.youtube.com` document / page
+  5. Copy the `Cookie` header from the associated response request
+  6. Create a file in your users home directory called `headers.txt`
```

Comment:
Thanks a lot for your contribution. I appreciate the dedication. 
The headers file needs to be in the same dir as the `ytermusic` binary. I should probably change that (I'm not sure the user will be able to find the config file if it's in the app configs folder).

Reply:
Hmm okay strange, I could have sworn I moved my binary to `/usr/local/bin/ytermusic` and had headers in my home dir. 

Ill update this. 

But the `.config` thing is a very common convention, people will definitely be familiar with it.

Reply:
Fixed